jekyll/_cci2/server/v4.4/installation/upgrade-server.adoc

```diff
@@ -39,7 +39,7 @@ We have moved away from Vault to Tink for encryption. The process for migration
 [#upgrade-steps]
 == Upgrade steps
 
-. Ensure your cluster is running a compatible Kubernetes version for this release (1.26 - 1.29).
+. Ensure your cluster is running a compatible Kubernetes version for this release (1.26 - 1.31).
```

Comment:
This update can be added to all affected versions

Reply:
Also done. :-)